sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/Azure.ResourceManager.DeviceRegistry.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.DeviceRegistry</PackageId>
+    <Description>Azure Resource Manager client SDK for Azure resource provider DeviceRegistry.</Description>
+    <PackageTags>azure;management;arm;resource manager;deviceregistry</PackageTags>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)ChangeTrackingList.cs" LinkBase="Shared/Core" />
```

Comment:
For reference: had to add L9-10 to fix `dotnet build` errors in the generated code. The errors look like:

- `error CS0122: 'Optional<T>.Value' is inaccessible due to its protection level`
- `error CS0122  'ChangeTrackingList<T>' is inaccessible due to its protection level`

Reply:
This has been fixed. Thanks @ArthurMa1978.